src/lib.rs

```diff
@@ -79,6 +56,7 @@ pub trait SerializerAcceptor {
 /// |------------|------------|
 /// | Unlimited  | Little     |
 #[inline(always)]
+#[deprecated(since = "1.3.0", note = "please use `DefaultOptions::new()` instead")]
 pub fn config() -> Config {
```

Comment:
Making this `fn config()` deprecated throws a lot of warnings in the tests. This should be fixed before this PR is merged (but after an API is settled)